parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -464,17 +464,8 @@ def execute_task(bufs):
 
     code = "{0} = {1}(*{2}, **{3})".format(resultname, fname,
                                            argname, kwargname)
-    try:
-        # logger.debug("[RUNNER] Executing: {0}".format(code))
-        exec(code, user_ns, user_ns)
-
-    except Exception as e:
-        logger.warning("Caught exception; will raise it: {}".format(e), exc_info=True)
-        raise e
-
-    else:
-        # logger.debug("[RUNNER] Result: {0}".format(user_ns.get(resultname)))
-        return user_ns.get(resultname)
+    exec(code, user_ns, user_ns)
```

Comment:
The exception was actually clobbered because ``logger`` wasn't defined. I thought the log message was not super useful anyway so I just removed it and the whole try/except block.